server/src/tools/daeml.cpp

```diff
@@ -12,11 +12,11 @@
 void closeall(int fd)
```

Comment:
I dont think this `daeml` is necessary as we are switching to use `docker-compose` or `kubernetes`

Reply:
I accidentally committed this file.
Agreed. With Docker/Kubernetes/systemd, home-brewed process supervisor is not needed.